### ISSUE TITLE
Add onClickBottomside method in View

### ISSUE THREAD

**Issue by luapascoal** [DevNatan/inventory-framework#105]
Refs: https://github.com/ScreenNetwork/inventory-framework-fork/blob/main/framework/src/main/java/me/saiintbrisson/minecraft/View.java (551)
https://github.com/ScreenNetwork/inventory-framework-fork/blob/main/framework/src/main/java/me/saiintbrisson/minecraft/ViewListener.java (150)

We removed a large majority of the listener and `handleMoveIn` method for this. But it's a useful thing for us and probably very useful for the framework itself. We use this method to detect clicked items in the player's inventory when he has a View open.

**Comment by DevNatan** [comment]
I understand the need to detect clicks outside the view's inventory, but the design of creating a new handler in the View class is not ideal from an API design point of view.

With each new handler that is created, the proliferation of code that is not coming from the user in that user's View class gets bigger and bigger, and, we already have `onClick` for that, so it would be better to use it in this case.

I can create a `ViewSlotContext#isViewContainer` option to detect if that context (in your click case) occurred inside the player container or in the view container and would be used like this:
```java
@Override
protected final void onClick(ViewSlotContext context) {
    if (!context.isViewContainer())
        // clicked on player inventory
}
```

Also, the code is in a rewrite process so the less need to inherit functions from the view, which will soon be deprecated, better.

**Comment by DevNatan** [comment]
Now available on [v2.5.1-rc.1](https://github.com/DevNatan/inventory-framework/releases/tag/2.5.1-rc.1)
